steal-qunit.js

```diff
@@ -111,8 +111,8 @@ define([
 		if (window.Testee && window.Testee.init) {
 			Testee.init();
 		}
-		
-		if(typeof QUnit.start === "function") {
```

Comment:
This is not the right check for qunit 2, this is vailable in both 1x and 2x
![image](https://cloud.githubusercontent.com/assets/3850812/25266727/cf348898-268f-11e7-9c02-36fc60e4a7ed.png)